README.md

```diff
@@ -157,12 +157,12 @@ Use the [`xrpl.transaction`](https://xrpl-py.readthedocs.io/en/stable/source/xrp
 
 * [`sign`](https://xrpl-py.readthedocs.io/en/stable/source/xrpl.transaction.html#xrpl.transaction.sign) — Signs a transaction locally. This method **does  not** submit the transaction to the XRP Ledger.
 
-* [`send_reliable_submission`](https://xrpl-py.readthedocs.io/en/stable/source/xrpl.transaction.html#xrpl.transaction.send_reliable_submission) — An implementation of the [reliable transaction submission guidelines](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission), this method submits a signed transaction to the XRP Ledger and then verifies that it has been included in a validated ledger (or has failed to do so). Use this method to submit transactions for production purposes.
+* `submit_and_wait` — An implementation of the [reliable transaction submission guidelines](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission), this method submits a signed transaction to the XRP Ledger and then verifies that it has been included in a validated ledger (or has failed to do so). Use this method to submit transactions for production purposes.
```

Comment:
Shouldn't this still link to the docs?

Reply:
`submit_and_wait` isn't in the docs yet - I have it as a follow up item to add the link later. (https://xrpl-py.readthedocs.io/en/stable/search.html?q=submit_and_wait&check_keywords=yes&area=default)

Reply:
The link will probably be the same if you just want to put it in for now.

Reply:
I'll just include it in a follow up PR afterwards (Have to do it in a couple other places anyway)